File: benchmark/comparison/results/haiku_py/02.py
def find_first_even(lst):
    for num in lst:
        if num % 2 == 0:
            return num
    return -1

# Test
print(find_first_even([1, 3, 2, 5]))
print(find_first_even([1, 3, 5]))
print(find_first_even([4, 2, 6]))

File: benchmark/comparison/results/haiku_ts/02.ts
function findFirstEven(lst: number[]): number {
  for (const num of lst) {
    if (num % 2 === 0) {
      return num;
    }
  }
  return -1;
}

// Test cases
console.log(findFirstEven([1, 3, 2, 5]));
console.log(findFirstEven([1, 3, 5]));
console.log(findFirstEven([4, 2, 6]));
